specs/windows/arp_cache.table

```diff
@@ -0,0 +1,15 @@
+table_name("arp_cache")
```

Comment:
For a virtual table like this, I think we'd want to find a way to get parity with the posix `arp_cache` table equivalent. So what we might need to do is leave out the extra data for the `arp_cache` table, bring the `arp_cache` spec out of the `linux` folder up one level for it to be cross-platform. Then possibly 

  1.) Extend the posix `arp_cache` tables to include IPv6, and 
  2.) potentially roll out another windows virtual table called `arp_cache_extended` that contains the additional columns of information. 

@theopolis what do you think?